covid_moonshot_ml/datasets/utils.py

```diff
@@ -93,6 +45,23 @@ def add_seqres(pdb_in, pdb_out=None):
     print(f"Wrote {pdb_out}", flush=True)
 
 
+def seqres_to_res_list(SEQRES_str):
+    """
+    https://www.wwpdb.org/documentation/file-format-content/format33/sect3.html#SEQRES
+    Parameters
+    ----------
+    SEQRES_str
+
+    Returns
+    -------
+
+    """
+    seq_lines = [line[19:] for line in SEQRES_str.split("\n") if " A " in line]
```

Comment:
is this for checking that the SEQRES is only for chain A? I think it might be better to manually check the column in the entry that gives the chain, just in case the `" A "` pattern somehow appears elsewhere in the entry